src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeExceptionHelpers.cs

```diff
@@ -192,6 +192,29 @@ public struct ExceptionInformationArray
         }
 
         private static ulong s_crashingThreadId;
+        private static CrashInfo s_crashInfo;
```

Comment:
```suggestion
        private static IntPtr s_triageBufferAddress;
        private static int s_triageBufferSize;
```
We do not need the keep the full CrashInfo writer around. We just need the pointer to the triage buffer.